sdks/python/apache_beam/runners/dataflow/dataflow_runner.py

```diff
@@ -312,6 +312,11 @@ def run_pipeline(self, pipeline):
     if apiclient._use_fnapi(pipeline._options):
       pipeline.visit(self.side_input_visitor())
 
+    # Performing configured PTransform overrides.  Note that this is currently
+    # done before Runner API serialization, since the new proto needs to contain
+    # any added PTransforms.
+    pipeline.replace_all(DataflowRunner._PTRANSFORM_OVERRIDES)
+
     # Snapshot the pipeline in a portable proto before mutating it
```

Comment:
nit, this comment doesn't make sense anymore

Reply:
Thanks.